pyfolio/plotting.py

```diff
@@ -1734,27 +1728,181 @@ def plot_multistrike_cones(is_returns, oos_returns, num_samples=1000,
             if crossing.sum() <= 0:
                 break
             cone_start = crossing.loc[crossing].index[0]
-            returns_tmp = oos_returns.loc[cone_start:]
+            returns_tmp = returns.loc[cone_start:]
             bounds_tmp = (bounds - (1 - returns.loc[cone_start]))
         for std in cone_std:
             x = returns_tmp.index
             y1 = bounds_tmp[float(std)].iloc[:len(returns_tmp)]
             y2 = bounds_tmp[float(-std)].iloc[:len(returns_tmp)]
             axes.fill_between(x, y1, y2, color=colors[c], alpha=0.5)
+
     # Plot returns line graph
-    returns.plot(
-        ax=axes,
-        lw=3.,
-        color='black',
-        label='Cumulative returns = {:.2f}%'.format(
-            (returns.iloc[-1] - 1) * 100)
-    )
+    label = 'Cumulative returns = {:.2f}%'.format((returns.iloc[-1] - 1) * 100)
+    axes.plot(returns.index, returns.values, color='k', lw=3.,
+              label=label)
+
     if name is not None:
         axes.set_title(name)
-    axes.axhline(1, color='black', alpha=0.2)
+    axes.axhline(1, color='k', alpha=0.2)
```

Comment:
I prefer `black` too, I just wanted to be consistent and changed this to match an earlier line. I'll change it to back to `black`.